### PR TITLE
roles: hosted_engine_setup: Remove cloud-init configuration

### PR DESCRIPTION
**Pull request by arachmani**
Remove cloud-init configuration from the engine VM

Bug-Url: https://bugzilla.redhat.com/1965456
Signed-off-by: Asaf Rachmani <arachman@redhat.com>

### CODE REVIEW

File: roles/hosted_engine_setup/tasks/create_target_vm/02_engine_vm_configuration.yml [hunk 1]
```diff
@@ -86,3 +86,6 @@
       regexp: "^\\s*PermitRootLogin"
       line: "PermitRootLogin {{ he_root_ssh_access }}"
       state: present
+  - name: Clean cloud-init configuration
+    include_tasks: ../clean_cloud_init_config.yml
+    ignore_errors: true
```
Comment (sandrobonazzola) [inline roles/hosted_engine_setup/tasks/create_target_vm/02_engine_vm_configuration.yml:91]: why do we need `ignore_errors: true` here?

**Review by sandrobonazzola — COMMENTED** [review]
just a question inline, the PR looks good to me otherwise.

File: roles/hosted_engine_setup/tasks/create_target_vm/02_engine_vm_configuration.yml (hunk 1 above)
Comment (arachmani) [reply roles/hosted_engine_setup/tasks/create_target_vm/02_engine_vm_configuration.yml:91]: We don't really need it, I'll remove that.

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Comment by arachmani** [comment]
Verified

**Pull request merged**